### ISSUE TITLE
[FR] Docker container - to preserve the OS

### ISSUE THREAD

**Issue by tubalainen** [jekhokie/raspberry-noaa-v2#378]
**Please Describe the problem to be solved**
To ease management, installation and upgrades as well as keeping the original OS intact, please consider creating a docker container for this fantastic hack! :)

Thanks for your hard and dedicated work! <3

**Comment by dom-robinson** [comment]
Good idea. I also think a ready to configure image might be a good thing to produce at some point. 

There is a bit of  a backlog of PRs and issues we are going to filter and prioritise over the next few weeks. Personally i think we need to debug a few things and add a few more features that have been waiting to come online for a while before we cut images / dockercontainers - else we will be adding more fire to the fire lol!

But i agree its a good idea - can you remind / bug us in a month or two?

**Comment by tubalainen** [comment]
Sure thing - will try to remember :)

Here is an example of a great implementation we maybe, it also uses RTL-SDR and got similar (basic) features to this project.

https://github.com/fredclausen/docker-acarshub

**Comment by tubalainen** [comment]
Gentle poke :)

I have been working with docker for a few years and its fantastic. I will try to take a initial stab at getting it going.
Anyone wanna collaborate? :)

//T

**Comment by dom-robinson** [comment]
Very happy to throw some input in - i did have a crack, but tbh it was distracting me so much from progressing the 'urgent' to do lists for RN2 i parked it and then realised i wiped that SD card (doh!)

I had more or less got it working but the external dockerfile was getting more complex to manage on each install and i worked out that going through the docker setup was taking longer than a clean SD card install, so for my own purposes of dev it was becoming counter-efficient. 

That said happy to throw thoughts in here if you progress and need some input!

**Comment by rhaamo** [comment]
A docker image would be really useful, right now I already have a dedicated server for radio stuff and I don't want to spend money on yet-another-pi and it's not possible to install at all without having to spend hours digging into the install part.

**Comment by sachasmart** [comment]
Watching this as well! Happy to give it a go if this has not progressed?
